### ISSUE TITLE
errors from time to time

### ISSUE THREAD

**Issue by jahlives** [gryphius/fuglu#89]
I'm using the latest fuglu version ``` 0.6.6-23-g20465de``` and from time to time mails get noqueue due to a python error I can find in the maillogs (added some linebreaks)
```
Jan 14 12:55:48 mail fuglu[10323]: Plugin SpamAssassin failed: Traceback (most recent call last):
#012  File "/usr/lib/python2.7/site-packages/fuglu/scansession.py", line 231, in run_plugins
#012    ans = plugin.examine(suspect)#012  File "/usr/lib/python2.7/site-packages/fuglu/plugins/sa.py", line 440, in examine#012    self.section, 'highspamlevel')
#012  File "/usr/lib/python2.7/site-packages/backports/configparser/__init__.py", line 841, in getfloat#012    return self._get_conv(section, option, float, **kwargs)#012  File "/usr/lib/python2.7/site-packages/backports/configparser/__init__.py", line 822, in _get_conv#012    return self._get(section, conv, option, **kwargs)
#012  File "/usr/lib/python2.7/site-packages/backports/configparser/__init__.py", line 814, in _get
#012    return conv(self.get(section, option, **kwargs))#012TypeError: get() got an unexpected keyword argument 'raw'
```
and some lines later in the logs I get this
```
Jan 14 12:55:48 mail fuglu[10323]: Could not commit message. Error: 'ascii' codec can't decode byte 0x92 in position 3170: ordinal not in range(128)
```
My system is a Centos7 with latest updates and spamassassin in version ```3.4.0-2.el7```

Any idea what might be the cause for this error?

Cheers

tobi

**Comment by gryphius** [comment]
at least one of the errors seems to be related to your `highspamlevel` setting
please post your configuration

**Comment by jahlives** [comment]
I did not read the error message well enough. After your remark I saw that configparser was mentioned in the error message as well. And that package I installed some days ago via pip and since then fuglu was not happy. I removed the configparser package from pip as I saw I have a ```python-iniparse 0.4-9``` package via yum.
Since I removed the additional configparser package the error does not show up anymore
